debian/changelog

```diff
@@ -1,3 +1,9 @@
+system76-driver (20.04.108~~alpha) focal; urgency=low
+
+  * Adds kernel revision to systeminfo.txt file
```

Comment:
You have to leave the "Daily WIP" item there and add yours below it. See https://github.com/pop-os/system76-driver/pull/315/files for reference.